src/HomeWorkApp.java

```diff
@@ -0,0 +1,175 @@
+import java.util.Scanner;
+
+public class HomeWorkApp {
+
+    public static void main(String[] args) {
+        printThreeWords();
+        checkSumSign();
+        printColor();
+        compareNumbers();
+        inRange();
+        plusMinus();
+        plusTrueMinusFalse();
+        numberOfTimes();
+        leapYear();
+    }
+
+    // Task 2.
+    public static void printThreeWords() {
+        System.out.printf("Orange%nBanana%nApple%n- - -%n");
+    }
+
+    // Task 3.
+    public static void checkSumSign() {
+        int a = -900;
+        int b = 150;
+        if ((a + b) >= 0) {
+            System.out.printf("The amount is positive%n- - -%n");
+        } else {
+            System.out.printf("The amount is negative%n- - -%n");
+        }
+    }
+
+    // Task 4.
+    public static void printColor() {
+        int value = 500;
+        if (value <= 0) {
+            System.out.printf("Red%n- - -%n");
+        } else if (value > 0 && value < 100) {
+            System.out.printf("Yellow%n- - -%n");
+        } else {
+            System.out.printf("Green%n- - -%n");
+        }
+    }
+
+    // Task 5.
+    public static void compareNumbers() {
+        int a = -900;
+        int b = 150;
+        if (a >= b) {
+            System.out.printf("a >= b%n- - -%n");
+        } else {
+            System.out.printf("a < b%n- - -%n");
+        }
+    }
+
+    // Task 6.
+    public static void inRange() {
+        int a;
+        int b;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Check sum a+b between 10 and 20");
+        System.out.println("Enter the value of a:");
+        a = sc.nextInt();
+        System.out.println("Enter the value of b:");
+        b = sc.nextInt();
+        System.out.printf("Result is %b%n- - -%n", checkRange(a, b));
+    }
+
+    public static boolean checkRange(int a, int b) {
+        if (a + b <= 20 && a + b >= 10) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    // Task 7.
+    public static void plusMinus() {
+        int a;
```

Comment:
don't separate the declaration and initialization of a variable, it will be better to declare it at line 82